src/main/java/pl/edu/icm/pl/mxrdr/extension/importer/xds/XdsImporterForm.java

```diff
@@ -0,0 +1,17 @@
+package pl.edu.icm.pl.mxrdr.extension.importer.xds;
+
+import edu.harvard.iq.dataverse.importer.metadata.ImporterFieldKey;
+
+public enum XdsImporterForm implements ImporterFieldKey {
+    XDS_FILE;
+
+    @Override
+    public String getName() {
```

Comment:
This method should return the unique name for each of the enum constants. In this case there is no problem, but in the case of more items it would work improperly.